src/server/host/Endpoints/Administration/Events/UpdateEventEndpoint.cs

```diff
@@ -17,14 +18,18 @@ namespace MinigolfFriday.Host.Endpoints.Administration.Events;
 // / <param name="Commit">set true if you want to commit the event and send a notification to the user</param>
 public record UpdateEventRequest(
     [property: Required] string EventId,
-    [property: Required] bool Commit
+    bool? Commit,
+    string? ExternalUri
 );
 
 public class UpdateEventRequestValidator : Validator<UpdateEventRequest>
 {
     public UpdateEventRequestValidator(IIdService idService)
     {
         RuleFor(x => x.EventId).NotEmpty().ValidSqid(idService.Event);
+        RuleFor(x => x)
```

Comment:
This rule is not really necessary. When not providing anything to update, the PATCH endpoint should just do nothing and return a 200.

Reply:
with this I was able to detect where my issue was when trying to clear the externalUri. without it, it just would not work. but now I was able to detect the issue inside the client.

What do you think? If actually nothing will be changed, is it really a 200?

Reply:
I guess this is debatable ^^
I like to return error codes only if there is actually something wrong. When not providing any updates, that is a valid request. It will just do nothing, but that is expected then.

Reply:
reasonable argument!